packages/popups/src/Notice/toastNoticeHooks/useBottomToastNoticeYSnapPoints.ts

```diff
@@ -3,6 +3,9 @@ import { useSafeAreaInsets } from 'react-native-safe-area-context';
 import { UIConstant } from '@tonlabs/uikit.navigation';
 import type { SnapPoints } from '../types';
 
+// @inline
+const INVALID_VALUE = 100000;
```

Comment:
This value is reused across several files. Can we move it at a single spot?

Reply:
It's `@inline`, so no :)

Reply:
Crap :)